ColorMyViews/app/src/main/java/com/android/example/colormyviews/MainActivity.kt

```diff
@@ -49,9 +50,9 @@ class MainActivity : AppCompatActivity() {
 
```

Comment:
Should the above findByViewId calls be with TextView or Button?

Reply:
No they are Textviews.
<TextView
        android:id="@+id/box_one_text"